test/test_aqua.jl

```diff
@@ -1,12 +1,15 @@
 module TestAqua
 
 using Aqua
+using ExplicitImports
```

Comment:
```suggestion
using ExplicitImports: check_no_implicit_imports, check_no_stale_explicit_imports
```

Reply:
Speaking of explicit imports... :sweat_smile: